assignment.js

```diff
@@ -113,6 +143,50 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+let playerAChoice = 'scissors'
+let playerBChoice = 'paper'
+
+// //array of possible choices
+// choices = ['paper','rock','scissors']
+//
+// //function to randomly choose index from choices array
+// function getRandomInt(max) {
+//   return Math.floor(Math.random() * Math.floor(max));
+// }
+//
+// let playerAChoice = choices[getRandomInt(3)]
+// let playerBChoice = choices[getRandomInt(3)]
+
+console.log(`playerA had ${playerAChoice} and playerB had ${playerBChoice}`)
```

Comment:
@dkwatson522 nice job providing additional output to communicate which options were selected by each player